db/migrate/20170215204213_requester_id_optional.rb

```diff
@@ -0,0 +1,5 @@
+class RequesterIdOptional < ActiveRecord::Migration[5.0]
+  def change
+    change_column_null :course_queue_entries, :requester_id, true
+  end
+end
```

Comment:
This is from a first pass implementation and should be removed. `requester_id` is still required. Also fix this in the `belongs_to` association.